### ISSUE TITLE
LiveReload regression in v3.5

### ISSUE THREAD

**Issue by josemarluedke** [ember-cli/ember-cli#8129]
With the release of Ember CLI v3.5, the live reload feature seem to be regressed.

The only error that I have seen is after a while with the page open in the browser, I get this error `Firefox can’t establish a connection to the server at ws://localhost:4200/_lr/livereload.`

I have created a new app to make sure this happens there as well and I can confirm that's the case.

https://github.com/josemarluedke/--livereload-repro

I have added `ember-cli-inject-live-reload@^2.0.0`, but no change.
I also played with `liveReloadPrefix` in `.ember-cli` and it did not solve the problem.

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.5.0
http_parser: 2.8.0
node: 10.11.0
v8: 6.8.275.32-node.28
uv: 1.23.0
zlib: 1.2.11
ares: 1.14.0
modules: 64
nghttp2: 1.33.0
napi: 3
openssl: 1.1.0i
icu: 62.1
unicode: 11.0
cldr: 33.1
tz: 2018e
os: darwin x64
6.4.1
1.10.1
```

**Comment by stefanpenner** [comment]
cc @sparshithnr

**Comment by chrism** [comment]
I'm having the same issue.

Let me know if anymore information would be useful.

**Comment by SparshithNR** [comment]
Please pin version of `ember-cli-inject-live-reload` [1.8.2](https://github.com/ember-cli/ember-cli-inject-live-reload/releases/tag/v1.8.2)

**Comment by SparshithNR** [comment]
Version check [here](https://github.com/ember-cli/ember-cli-inject-live-reload/blob/master/index.js#L59) in `ember-cli-inject-live-reload` is causing this issue. `ember-cli` [PR](https://github.com/ember-cli/ember-cli/pull/8096) which I had, couldn't make it to `3.5.0` version of `ember-cli`

**Comment by stefanpenner** [comment]
We will land @SparshithNR fix, in a fast follow up to 3.5.x.

**Comment by josemarluedke** [comment]
I can confirm that version 1.8.2 of `ember-cli-inject-live-reload` works.

**Comment by rwjblue** [comment]
Should be fixed by https://github.com/ember-cli/ember-cli-inject-live-reload/pull/66 which was released (along with another fix) in 2.0.1 and 1.10.2.
